Competition2017/src/competition/subsystems/shooter/SideShooterSubsystem.java

```diff
@@ -0,0 +1,167 @@
+package competition.subsystems.shooter;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SideShooterSubsystem extends BaseSubsystem {
+    
+    private static Logger log = Logger.getLogger(ShooterSubsystem.class);
+    
+    private final XCANTalon masterMotor;
+    public final String side;
+    
+    // output telemetry properties
+    private final DoubleProperty shooterCurrentSpeed;
+    private final DoubleProperty shooterTargetSpeed;
+    private final DoubleProperty shooterOutputPower;
+    private final DoubleProperty shooterTalonError;
+    private final BooleanProperty atSpeedProp;
+    
+    //private final DoubleProperty encoderCodesProperty;
+    private final DoubleProperty p;
+    private final DoubleProperty i;
+    private final DoubleProperty d;
+    private final DoubleProperty f;
+    
+    private final DoubleProperty shooterSpeedThresh;
+    
+   
+    public SideShooterSubsystem(int motor, String side,WPIFactory factory, XPropertyManager propManager){
+        log.info("Creating" + side + " shooter");
+        this.side = side;
+        
+        p = propManager.createPersistentProperty(side + "FuelLauncher wheel P", 0);
+        i = propManager.createPersistentProperty(side + "FuelLauncher wheel I", 0);
+        d = propManager.createPersistentProperty(side + "FuelLauncher wheel D", 0);
+        f = propManager.createPersistentProperty(side + "FuelLauncher wheel F", 0);
+        
+        // Reading through the manual, it looks like only a few things support human units. It might be easier to just keep everything in native units per 100ms,
+        // which is what the Talon uses for all its calculations.
+        // I call this Ticks per Centisecond, or TPC
```

Comment:
There's lots of duplicated stuff here (from the fuel launcher) and most refers to the name of the other subsystem. Which is supposed to exist and which should be removed? I'd assume this is the correct one, in which case my comments on the other apply here.